packages/react-menu/src/components/MenuTrigger/useTriggerElement.ts

```diff
@@ -31,18 +32,11 @@ export const useTriggerElement = (state: UseTriggerElementState): MenuTriggerSta
     child.props?.onContextMenu?.(e);
   });
 
-  const onMouseEnter = useEventCallback((e: React.MouseEvent) => {
+  const onFocus = useEventCallback((e: React.MouseEvent) => {
```

Comment:
I don't think that it's a relevant replacement as it breaks `onHover` behavior... IMO, we should still handle `onMouseEnter`:

![menu-hover](https://user-images.githubusercontent.com/14183168/112345360-404a7700-8cc5-11eb-8159-df71da12e671.gif)

Reply:
Yeah close on blur made sense, but open on focus not really.... replace `onFocus` with `onMouseEnter`